src/pages/view/FilterBar/styles/FilterBadge.module.css

```diff
@@ -22,12 +22,29 @@
 }
 
 .removeButton {
+    background-color: transparent;
+    color: white;
```

Comment:
```suggestion
    color: var(--colors-white);
```

Reply:
applies to a few other places as well. maybe just do a search and replace?

Reply:
Fixed here. I'll do a repo wide fix in a separate PR.